packages/cli/test/unit/commands/status.unit.test.ts

```diff
@@ -52,6 +50,17 @@ Tools:     No known issues at this time.\n`)
 })
 
 describe('when heroku has issues', () => {
+  const now = Date.now()
+  const timeISO = new Date(now).toISOString()
+
+  before(() => {
+    sinon.stub(Date, 'now').returns(now)
+  })
+
+  after(() => {
+    sinon.restore()
+  })
+
```

Comment:
There was a hidden race condition buried in this test. I think the way Lerna runs test concurrently is more apparent when running on a larger test suite in a single package.

Reply:
Good catch! Those can be tough to find.